lib/md007.js

```diff
@@ -12,11 +12,13 @@ module.exports = {
   "tags": [ "bullet", "ul", "indentation" ],
   "function": function MD007(params, onError) {
     const optionsIndent = params.config.indent || 2;
+    const optionsLevel1Indent = (params.config.first_level_indented === true);
```

Comment:
So, I renamed the var (I'd had that different name just because of the length, which we solved by shortening the config param name).

But, I didn't make it use loose equality. Since there's a schema that defines the config as undefined or boolean, I figured it was safe / better to leave it as strict equality. But, let me know if you prefer the loose equality and type coercion that goes along with it.

Reply:
Here is the pattern I prefer to use: https://github.com/DavidAnson/markdownlint/blob/5634b317d6e152e2708c0cc1ecd12467101a9edb/lib/md013.js#L32

I see now that I wasn’t entirely consistent here because the existing variable has an “options” prefix. If you could please update that when you make this update, I would greatly appreciate it.